src/README.md

```diff
@@ -0,0 +1,29 @@
+# Run Tribler
+
+Full instruction: [install requirements](https://github.com/Tribler/tribler#setting-up-your-development-environment)
+
+```
+python3 -m pip install -r requirements.txt
+
+tribler.sh
+```
+
+# Run Tests
+
+Install all necessary dependencies:
+
+```
+python3 -m pip install -r requirements-test.txt
```

Comment:
Nitpick: from the description it is not clear, does `requirements-test.txt` include dependencies from `requirements.txt`, or dependences from `requirements.txt` should be installed as well.

Reply:
Fixed